src/lib.rs

```diff
@@ -259,7 +261,8 @@ impl Window {
 
     /// Deletes the window, freeing all associated memory. In the case of overlapping windows,
     /// subwindows should be deleted before the main window.
-    pub fn delwin(self) -> i32 {
+    pub fn delwin(mut self) -> i32 {
+        self._deleted = true;
         unsafe { curses::delwin(self._window) }
```

Comment:
rather than tracking the state and using a boolean, you probably could have just used `std::mem::forget` here, I think. I could be wrong.

Reply:
but I guess that would leak some small amount of memory. I wonder if there's a way to do `mem::drop` and avoid running the `Drop` impl?